src/components/pages/Login/LoginContainer.js

```diff
@@ -12,16 +12,18 @@ const LoginContainer = () => {
       baseUrl: issuer ? issuer.split('/oauth2')[0] : '',
       clientId,
       redirectUri,
-      registration: {
+      tration: {
```

Comment:
This looks suspicious

Reply:
Nice catch!  I don't know what happened there.